src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -157,15 +157,15 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
           val response = WorkspaceListResponse(accessLevel,
             workspaceContext.workspace,
             getWorkspaceSubmissionStats(workspaceContext, txn),
-            getWorkspaceOwners(workspaceName, txn))
+            getWorkspaceOwners(workspaceContext.workspace, txn))
           Future.successful(RequestComplete(StatusCodes.OK, response))
         }
       }
     }
 
-  def getWorkspaceOwners(workspaceName: WorkspaceName, txn: RawlsTransaction): Seq[String] = {
-    val ownerGroup = containerDAO.authDAO.loadGroup(RawlsGroup(workspaceName, WorkspaceAccessLevels.Owner), txn).getOrElse(
-      throw new RawlsException(s"Unable to load owners for workspace ${workspaceName}"))
+  def getWorkspaceOwners(workspace: Workspace, txn: RawlsTransaction): Seq[String] = {
+    val ownerGroup = containerDAO.authDAO.loadGroup(workspace.accessLevels(WorkspaceAccessLevels.Owner), txn).getOrElse(
+      throw new RawlsException(s"Unable to load owners for workspace ${workspace.toWorkspaceName}"))
```

Comment:
the old way looked up the owners group by name, the new way follows the edge reference which is resilient to naming convention changes